### PR TITLE
feat(i18n): manual routing

### PR DESCRIPTION
**Pull request by ematipico**
#### Description (required)

Closes PLT-1921

This PR adds documentation for the upcoming i18n manual routing.

 #### For Astro version: `4.6.0`. See astro PR [#10193](https://github.com/withastro/astro/pull/10193).

### CODE REVIEW

**Review by ematipico — COMMENTED** [review]
The `manual` configuration paragraph looks good to me

File: src/content/docs/en/guides/internationalization.mdx [hunk 1]
```diff
@@ -456,3 +537,10 @@ console.log(getLocaleByPath("portugues")) // will log "pt-AO"
 
 [`site`]: /en/reference/configuration-reference/#site
 [`i18n.locales`]: /en/reference/configuration-reference/#i18nlocales
+
+
+### `redirectToDefaultLocale`
```
Comment (sarah11918) [inline src/content/docs/en/guides/internationalization.mdx:542]: @ematipico We're going to need some API reference for these new things! Can you take a first shot at adding reference descriptions for these?

File: src/content/docs/en/guides/internationalization.mdx [hunk 2]
```diff
@@ -1,98 +1,101 @@
...
 
-Astro's built-in file-based routing automatically creates URL routes for you based on your file structure within `src/pages/`. When you configure i18n routing, the `routing` value now allows you to specify your file structure (and corresponding URL paths generated) in order to use helper functions to generate, use, and verify the routes in your project.
+You can even choose to [implement your own routing logic manually](#manual) for even greater control.
+
+### `prefixDefaultLocale`
+
+This routing strategy defines whether or not your default language's URLs should use a language prefix (e.g. `/en/about/`).
+
+All non-default supported languages **will** use a localized prefix (e.g. `/fr/` or `/french/`) and content files must be located in appropriate folders. This configuration option allows you to specify whether your default launguage should also follow a localized URL structure.
```
Comment (yanthomasdev) [inline src/content/docs/en/guides/internationalization.mdx:92]: typo
```suggestion
All non-default supported languages **will** use a localized prefix (e.g. `/fr/` or `/french/`) and content files must be located in appropriate folders. This configuration option allows you to specify whether your default language should also follow a localized URL structure.
```

**Review by yanthomasdev — COMMENTED** [review]
Looking good so far, found just a small typo

File: src/content/docs/en/guides/internationalization.mdx (hunk 1 above)
Comment (ematipico) [reply src/content/docs/en/guides/internationalization.mdx:542]: sure, will do

**Comment by yanthomasdev** [comment]
> Woo! Links passed! Gonna need to i18nIgnore this one, unless @yanthomasdev wants something done differently here!

Not `i18nIgnore`! We gonna need a tracker directive for this one.

Please paste this into the merge commit body:
```
@lunaria-track:src/content/docs/en/guides/internationalization.mdx;src/content/docs/en/reference/api-reference.mdx;src/content/docs/en/reference/configuration-reference.mdx
@tracker-major:./src/content/docs/en/guides/internationalization.mdx;./src/content/docs/en/reference/api-reference.mdx;./src/content/docs/en/reference/configuration-reference.mdx;src/content/docs/en/guides/internationalization.mdx;src/content/docs/en/reference/api-reference.mdx;src/content/docs/en/reference/configuration-reference.mdx
```

**Comment by sarah11918** [comment]
Thanks @yanthomasdev ! Fortunately, this i18n page was never `i18n:true` in the first place, so the disruption to the MAIN page changed here is minimal. (But, KO translated it anyway?) So, I'll also convert this to true when it's merged. UNLEASH THE TRANSLATORS!

**Pull request merged**